test-source/test/ceylon/time/testDates.ceylon

```diff
@@ -141,7 +141,7 @@ shared test void testDateMinusMonths() {
 }
 
 shared test void testDateMinusLessDays() {
-    assertEquals( date(1982,november,30), date(1982,december,31).minusMonths(1));
+    assertEquals( date(1982,december,31).minusMonths(1), date(1982,november,30) );
```

Comment:
What about the test methods with more than one `assertEquals`? That would be a lot more lines…